docker/worker/Dockerfile

```diff
@@ -34,13 +33,19 @@ RUN pip3 install docker-explorer
 # Install Plaso
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0x5e80511b10c598b8 \
 && add-apt-repository -y ppa:gift/stable
-RUN apt-get -y install python-plaso \
+
+RUN apt-get update
+RUN apt-get -y install \
+libewf \
+python3-dfvfs \
+libewf-python3 \
+python3-plaso \
 plaso-tools \
-docker-explorer-tools \
```

Comment:
Why remove this?

Reply:
as far as I can see it's pip installed above?

Reply:
Yeah, interesting.  I remember now that Jason had added this here (#705)  because for some reason the worker wasn't finding the version installed from pip.  My guess is that his issue was related to sudo changing paths during execution, etc.  Since this is the one that seems to work, can we maybe remove the one from pip instead?

Reply:
Moved to apt instead of pip. Had a small discussion in the team and it makes sense to do a cleanup of the Docker files. Eg we are installing dfvfs twice. 

Going to do some more work on this, test the images and will ping you here when done.